packages/libp2p/src/circuit-relay/server/index.ts

```diff
@@ -95,9 +101,19 @@ export interface RelayServerEvents {
   'relay:advert:error': CustomEvent<Error>
 }
 
-const defaults = {
-  maxOutboundStopStreams: MAX_CONNECTIONS
-}
+const configValidator = object({
+  hopTimeout: number().min(0).integer().default(DEFAULT_HOP_TIMEOUT),
```

Comment:
probably should pick an order and stick with it.
Either `number().min(x).integer()`
or
`number().integer().min(x)`

my preference to the latter

Reply:
The latter reads better to me.